src/merkle.rs

```diff
@@ -2031,6 +2058,9 @@ where
     Ok(())
 }
 
+#[cfg(test)]
+use typenum::{U1, U11, U16, U4};
+
```

Comment:
That's totally OK, just FYI, you can also put the `use typenum::{U1, U11, U16, U4};` directly into the `test_get_merkle_tree_methods()`.

Reply:
Yes. It's just a manner of taste, I think, isn't it?